src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector512.cs

```diff
@@ -1390,13 +1405,7 @@ public static Vector512<T> CreateScalarUnsafe<T>(T value)
         /// <exception cref="NotSupportedException">The type of <paramref name="left" /> and <paramref name="right" /> (<typeparamref name="T" />) is not supported.</exception>
         [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static Vector512<T> Divide<T>(Vector512<T> left, Vector512<T> right)
-        {
-            return Create(
-                Vector256.Divide(left._lower, right._lower),
-                Vector256.Divide(left._upper, right._upper)
-            );
-        }
+        public static Vector512<T> Divide<T>(Vector512<T> left, Vector512<T> right) => left / right;
```

Comment:
Cases like this were where size regressions were coming from. This ensures that instead we hit the intrinsic path when we're accelerated and otherwise we get the same overall codegen as before.